python/lsst/cell_coadds/_fits.py

```diff
@@ -0,0 +1,407 @@
+# This file is part of cell_coadds.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+__all__ = (
+    "CellCoaddFitsFormatter",
+    "CellCoaddFitsReader",
+    "writeExplodedCoaddAsFits",
+    "writeMultipleCellCoaddAsFits",
+    "writeStitchedCoaddAsFits",
+)
+
+import os
+from typing import TYPE_CHECKING, Any, Mapping
+
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import numpy as np
+from astropy.io import fits  # type: ignore[import]
```

Comment:
I'd prefer for this ignore to go in the mypy.ini file, as another exception to the modules we expect to find types for.